community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md

```diff
@@ -0,0 +1,129 @@
+
+The following example shows how a sensor reading can be processed from a message handler implementation.
+```csharp
+using Arcus.Messaging.Abstractions.EventHubs.MessageHandling;
+using Microsoft.Extensions.Logging;
+
+public class SensorReadingAzureEventHubsMessageHandler : IAzureEventHubsMessageHandler<SensorReading>
+{
+    private readonly ILogger _logger;
+
+    public SensorReadingAzureEventHubsMessageHandler(ILogger<SensorReadingAzureEventHubsMessageHandler> logger)
+    {
+        _logger = logger;
+    }
+
+    public async Task ProcessMessageAsync(
+        SensorReading reading,
+        AzureEventHubsMessageContext messageContext,
+        MessageCorrelationInfo correlationInfo,
+        CancellationToken cancellation)
+    {
+        _logger.LogInformation("Process sensor reading {SensorId} (Type: {SensorType})", reading.SensorId, reading.SensorType);
+
+        // Process sensor reading...
+    }
+}
+```
+
+Registering this implementation is also very similar to Service Bus. We provide an `.AddEventHubsMessagePump(...)` extension to register a message pump implementation that retrieves events from EventHubs without any additional action on your part. The consumers' job is only focusing on how events, or in this case sensor reading events, should be processed.
+```csharp
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+Host.CreateDefaultBuilder()
+    .ConfigureSecretStore(...)
+    .ConfigureServices(services =>
+    {
+        services.AddEventHubsMessagePump("<event-hubs-name>", "<event-hubs-connection-string-secret-name>", "<blob-container-name>", "<storage-account-connection-string-secret-name>")
+                .WithEventHubsMessageHandler<SensorReadingAzureEventHubsMessageHandler, SensorReading>();
+    });
+```
+
+⚠ Note that the EventHubs connection string and storage account connection string are not passed along directly to the message pump extension, but that a secret name is provided. This secret name should correspond with a stored secret in the Arcus secret store. For more information on the secret store, see [our official documentation](https://security.arcus-azure.net/features/secret-store).
+
+💡 The EventHubs message pump and message router have a lot more options to influence the message handling to your needs. See [our official documentation](https://messaging.arcus-azure.net/Features/message-handling/event-hubs) for more information on this and other topics.
```

Comment:
```suggestion
💡 The EventHubs message pump and message router have a lot more options to fine tune the message handling to your needs. See [our official documentation](https://messaging.arcus-azure.net/Features/message-handling/event-hubs) for more information on this and other topics.
```

Reply:
It's really influence. It has deserialization options that could even alter the message handler that could process the message.

Reply:
What about modify ?

Reply:
https://github.com/arcus-azure/arcus/pull/246/commits/40c47259b54adb87b3a55c4bf2eb327bbeb48f95

Reply:
Changed it to 'modify'! 👍